From 9ca342a5c238872303410f4a99fcc84ea71b4104 Mon Sep 17 00:00:00 2001
From: Robert Dabrowski <robertx.dabrowski@intel.com>
Date: Fri, 28 Aug 2020 00:40:49 +0200
Subject: [PATCH] PONRTSYS-7795: Error messages from mcc driver during BBF
 6.3.x tests  + do not show error if entry not found in the table

---
 drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c b/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c
index 8c5f8ff1e958..35b3dfa584ec 100644
--- a/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c
+++ b/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c
@@ -11929,7 +11929,7 @@ GSW_return_t GSW_MulticastTableEntryRemove(void *cdev,
 			ret = gsw_remove_hashtable_entry(cdev, parm);
 
 			if (ret < 0) {
-				pr_err("%s:%s Entry Cannot remove :%d %d\n", __FILE__, __func__, __LINE__, ret);
+				pr_debug("%s:%s Entry Cannot remove :%d %d\n", __FILE__, __func__, __LINE__, ret);
 				ret = GSW_statusErr;
 				goto UNLOCK_AND_RETURN;
 			}
